### PR TITLE
chore: simplifying docker environment + syntax

### PR DESCRIPTION
**Pull request by Grraahaam**
What's fixed? 💡 
- Removed the `/docker` directory 🗑️ 
- Moved the custom images directly into the `docker-compose.yml` 🔽 
  - Since it just configuration through volumes + env variables
- Implemented volumes in `docker-compose.yml` 🗄️ 
  - In that case, `/storage` might now be useless 🤷🏽‍♂️ 
- Some syntax in the `Dockerfile` + `docker-compose.yml`
- Own your license, it's initially your work 👷🏽 

Cheers 🍺

### CODE REVIEW

**Comment by Grraahaam** [comment]
@AgRenaud You might want to put the name you want in the LICENSE, I've just put my buddy John meanwhile ✌🏽

**Comment by AgRenaud** [comment]
> * Implemented volumes in `docker-compose.yml` 🗄️
>   
>   * In that case, `/storage` might now be useless 🤷🏽‍♂️

In fact the scripts and notebook were using `storage/` folder (to store models before sharing it with container for exemple), I've change `docker-compose.yml` to fix this. It is easier for now to work this way.

* I've add Contributors to readme
* I've add my name to the Licence (thanks for the tip)

And I agree with all the other change thanks for taking time to take a look to the repo !

I'm merging the change

**Review by AgRenaud — APPROVED** [review]
LGTM ! 🎉

File: docker-compose.yml [hunk 1]
```diff
@@ -26,14 +26,14 @@ services:
...
-      - tf-data:/models/my_production_model/1/
+      - ./storage/models/my_production_model:/models/my_production_model/1/
 
   qdrant:
     image: generall/qdrant
     environment:
       RUST_BACKTRACE: "1"
     volumes:
-      - qdrant-data:/qdrant/storage
+      - ./storage/qdrant/storage:/qdrant/storage
```
Comment (Grraahaam) [inline docker-compose.yml:36]: Alright, if it's more convenient for you, then feel free to remove the below `volumes` section ✌🏽

**Pull request merged**